utils/_context/_scenarios/__init__.py

```diff
@@ -949,6 +949,14 @@ class _Scenarios:
         scenario_groups=[scenario_groups.appsec],
     )
 
+    ato_sdk = EndToEndScenario(
+        "ATO_SDK",
+        weblog_env={"DD_APPSEC_ENABLED": "false"},
+        doc="Test ato when appsec is diabled",
+        github_workflow="endtoend",
+        scenario_groups=[scenario_groups.end_to_end],
```

Comment:
It is testing when appsec is disabled, but I think that this hsould be also in `scenario_groups.appsec`  group, wdyt? It is an appsec feature.

Reply:
I will add it. I was in doubt